index.css

```diff
@@ -504,3 +567,370 @@ textarea {
+    font-weight: 350;
+    font-size: 19px;
+    line-height: 26px;
+  }
+
+  .box2 {
+    height: 435px;
+    background: url(../Desktop/Placeholder.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box3 {
+    height: 435px;
+    background: url(../Desktop/Placeholder4.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box4 {
+    height: 435px;
+    background: url(../Desktop/Placeholder5.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box {
+    height: 435px;
+    background: url(../Desktop/Placeholder3.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .dashbord1 {
+    display: none;
+  }
+
+  .texto1 {
+    display: none;
+  }
+
+  .contenido {
+    display: grid;
+    grid-column-gap: 20px;
+    background: url(Desktop/gymfit.png);
+    background-repeat: no-repeat;
+    background-position: 21% 0%;
+  }
+
+  .multi {
+    height: 48px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .brian1 button {
+    width: 369px;
+    height: 60px;
+    margin-top: 294px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    background-color: #ff6b00;
+    font-size: 17px;
+    color: #fff;
+  }
+
+  .brian button {
+    display: none;
+  }
+
+  .dashbord {
+    margin-top: 130px;
+  }
+
+  .dashbord2 {
+    margin-top: 130px;
+  }
+
+  .texto3 {
+    width: 337px;
+    height: 50px;
+    margin: 0;
+  }
+
+  .botones3 button {
+    width: 95px;
+    height: 40px;
+  }
+
+  .about {
+    background: #1c1a19 url(Desktop/Frame.png) 100% 40% no-repeat;
+    margin-top: 80px;
+    height: 610px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .about h3 {
+    width: 466px;
+    height: 78px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 88px;
+    color: #ff6b00;
+    padding-top: 90px;
+  }
+
+  .about p {
+    width: 566px;
+    height: 148px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin: 30px 0 0 90px;
+  }
+
+  .proyect2 button {
+    width: 161px;
+    height: 48px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    background: #ff6b00;
+    color: #fff;
+    margin-left: 90px;
+  }
+
+  .experience {
+    display: grid;
+    grid-template-columns: repeat(3, 400px);
+    height: 500px;
+  }
+
+  .letras2 {
+    margin-left: 20px;
+  }
+
+  .agrupador {
+    text-align: start;
+  }
+
+  .agrupador2 {
+    text-align: start;
+  }
+
+  .agrupador3 {
+    text-align: start;
+    background-position: 8% 6%;
+  }
+
+  .formato {
+    height: 390px;
+    margin: 80px 0 0 0;
+  }
```

Comment:
- [ ] Kindly add the right width to the first two input fields in your desktop design and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160049781-3525d038-c0a4-485f-8418-e0f349b3da8f.png) | ![image](https://user-images.githubusercontent.com/68796232/160049797-8e4e6193-f194-46d6-899c-fe0d40f3cefb.png)  |

Reply:
brian1 button has a class named see there i added the hover in my windows its showing the button change color when i clicked

Reply:
inactiv
![hover2](https://user-images.githubusercontent.com/74432345/160189854-a0848934-06a8-4a31-b0ce-ecab35d486c6.JPG)
e

when i click
![hover1](https://user-images.githubusercontent.com/74432345/160189978-fdf8c1e9-fc39-489f-8852-f929e7edef9c.JPG)

Reply:
IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

i think you mean the bottom line going to change it

Reply:
thank you for your CR